### PR TITLE
New TOML config and new Config cmd

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

File: cmd/exo/cmd/root.go [hunk 1]
```diff
@@ -57,18 +72,15 @@ func buildClient() {
...
 }
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+
 	envs := map[string]string{
-		"CLOUDSTACK_CONFIG": "config",
-		"CLOUDSTACK_REGION": "region",
+		"EXO_CONFIG":  "config",
+		"EXO_ACCOUNT": "account",
```
Comment (greut) [inline cmd/exo/cmd/root.go]: to be consistent, use the full `EXOSCALE` name

File: cmd/exo/cmd/root.go [hunk 2]
```diff
@@ -78,46 +90,64 @@ func initConfig() {
 		}
 	}
 
-	envEndpoint := os.Getenv("CLOUDSTACK_ENDPOINT")
-	envKey := os.Getenv("CLOUDSTACK_KEY")
-	envSecret := os.Getenv("CLOUDSTACK_SECRET")
+	envEndpoint := os.Getenv("EXO_ENDPOINT")
+	envKey := os.Getenv("EXO_KEY")
+	envSecret := os.Getenv("EXO_SECRET")
```
Comment (greut) [inline cmd/exo/cmd/root.go]: `EXOSCALE_API_KEY`, `EXOSCALE_API_SECRET`

Comment (greut) [reply cmd/exo/cmd/root.go]: we have many endpoints.

File: cmd/exo/cmd/root.go (hunk 1 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/root.go]: 👍

File: cmd/exo/cmd/root.go (hunk 2 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/root.go]: yes

File: cmd/exo/cmd/config.go [hunk 3]
```diff
@@ -395,6 +421,23 @@ func askQuestion(text string) bool {
 	return (strings.ToLower(resp) == "y" || strings.ToLower(resp) == "yes")
 }
 
+func checkCredentials(account *account) (*egoscale.Account, error) {
+	cs := egoscale.NewClient(account.Endpoint, account.Key, account.Secret)
+
+	resp, err := cs.Request(&egoscale.ListAccounts{})
+	if err != nil {
+		return nil, err
+	}
```
Comment (greut) [inline cmd/exo/cmd/config.go]: `List` ?

File: cmd/exo/cmd/config.go [hunk 4]
```diff
@@ -22,6 +23,13 @@ const (
 	exoConfigFileName = "exoscale"
 )
 
+var zones = []string{
+	"ch-gva-2",
+	"ch-dk-2",
+	"at-vie-1",
+	"de-fra-1",
```
Comment (greut) [inline cmd/exo/cmd/config.go]: use `listZones`

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/config.go]: yes

**Comment by pierre-emmanuelJ** [comment]
👍

**Pull request merged**